xsnap/makefiles/lin/xsnap-worker.mk

```diff
@@ -33,7 +31,7 @@ C_OPTIONS = \
 	-DmxLockdown=1 \
 	-DmxMetering=1 \
 	-DmxDebug=1 \
-	-UmxInstrument \
+	-DmxInstrument=1 \
```

Comment:
I believe this used to cause issues, or at least I wasn't confident in its behavior. See https://github.com/agoric-labs/xsnap-pub/pull/34

Reply:
As a rule `mxInstrument` and `mxDebug` are checked to see if they defined. The value shouldn't matter. That said, I prefer setting it to 1. But, whatever you do, don't set it 0 which suggests the feature is disabled when it isn't.

Reply:
`mxInstrument` is indeed still problematic and I’ve restored the `-U` flag just to be sure.